make_openbazaar.sh typo fix for bash and fish shells, remove brew call and added wine as required dep in README.md

M0Rf30 opened the pull request:
fish says
Failed to execute process './make_openbazaar.sh'. Reason:

exec: Exec format error

The file './make_openbazaar.sh' is marked as an executable but could not be run by the operating system.

bash says;
make_openbazaar.sh: riga 1: #!/bin/sh: File o directory non esistente

Commit:
converted with dos2unix

Commit:
don't use brew as default, instead show a warning

Commit:
Show wine as dep, cause of multiple wine version

Commit:
removed use of sudo, pyinstaller git branch (fix an error for ssl_match_host), use of png icon for linux, instead of OSX icns

Commit:
using local node_modules, do not interact system-widely in order to avoid conflicts with distro packages

Commit:
updated deps in README

M0Rf30 commented:
Refactor make_openbazaar.sh in order to be distro agnostic

M0Rf30 commented:
Perhaps there will be some errors for OSX building process, but I can't test

Commit:
deps checker

Commit:
deps checker

Commit:
fix makensis warning: Variable StrNoUsablePythonFound not referenced or never set, wasting memory!

Commit:
new lines

Commit:
updated python2

Commit:
added psutil

Commit:
added node for win

Commit:
removed icon for standalone linux exec

Commit:
working upx compression, pyinstaller-dev arguments fixed

Commit:
simplified git cloning (need many checks)

Commit:
trial to fix python modules nsis packaging

Commit:
trial to fix python modules nsis packaging

Commit:
win64 fixes

Commit:
win64 fixes

Commit:
added google-apputils

Commit:
software version as global var

Commit:
fix shebang

Commit:
trial with npm

Commit:
unzip overwrite

Commit:
unzip overwrite

Commit:
syntax error

Commit:
nodejs PATH setted in installer

Commit:
setting PYTHONPATH and PYTHONHOME global vars

Commit:
fix python executable and OpenBazaar path

Commit:
version downgrade

Commit:
gui started on windows :D, now try to slim package

Commit:
systray clean

Commit:
systray fixes

M0Rf30 committed:
Update systray.py

Commit:
temporary changed Server git repo

Commit:
use noupnp branch for server in win

Commit:
OpenBazaar now fully runs on win32/64

Commit:
git pull of repos

Commit:
firewall rules for python

Commit:
typo error

Commit:
typo error

Commit:
reworking linux and macos building process

Commit:
separate temp build folders

Commit:
separate temp build folders

Commit:
separate build folders

Commit:
separate build folders

M0Rf30 commented:
Instead of accept my pull request and fix macos errors compilation after, you think could be better read the suggestions and make my fork unusable for merge?

hoffmabc commented:
Sorry if I am misunderstanding you @M0Rf30 ...I don't understand the question. I am going to read through and try to help resolve.

hoffmabc commented on make_openbazaar.sh:
```diff
@@ -1,38 +1,34 @@
-﻿﻿#!/bin/sh
+#!/bin/bash
```
Why not just use the system's shell. Are there things here that are Bash specific? And in most cases sh will resolve to bash anyways.

hoffmabc commented:
OK I think I understand that you are agitated because I broke your PR with some later commits. I didn't realize you even had this PR submitted since it was a few days ago and I wasn't around. Apologies. I will try and pull down your branch and figure out how to clean it up for you.

hoffmabc commented:
OK I resolved the conflicts and merged. Sorry for being confusing.

M0Rf30 commented:
You are a sir, I'm happy to contribute to this futuristic project. My tones were not dictated by anger or ambition, but truly spirit of partecipation.

M0Rf30 commented:
Could you take a look at this?
https://aur.archlinux.org/packages/?O=0&K=openbazaar

This is my work of packaging for ArchLinux AUR. All the variants are fully working since your first works
Archlinux has a different method of packaging, more distro-agnostic respect to the most popular distros.

yagoulas commented:
Hello, there is something really weird with this pull request. Can you clarify that all these commits were meant to be submitted?

M0Rf30 commented:
I'm cleaning some errors regarding the packaging and extra data not needed by a well-done package. I've just collaborated to rewrite the make_openbazaar script. Now I'm just refining my work. The commits speak clearly

yagoulas commented:
Can you recheck the list of the commits that were merged? The list is huge and you have several changes that are related to windows and you don't mention them anywhere. The list of commits: https://github.com/OpenBazaar/OpenBazaar-Installer/pull/7/commits

M0Rf30 commented:
for ob.nsi:
I've basically cleaned nodejs refs not required
setted PYTHON global variables in win environment
setted python.exe in win firewall in order to obtain first runnable openbazaar session
removed an unneeded variable in nsis files
splitted win64 and win32 packages
used noupnp branch for server, cause of miniupnpc not compiled yet on win OS
used upx compression properly
used correct python executable in systray
changed pyinstaller to git release cause of "match_ssl_hostname" error

yagoulas commented:
Can you join slack to discuss this?

M0Rf30 commented:
How. I've never used slack. Invitation needed?

hoffmabc commented:
@M0Rf30 go here for an invite. https://openbazaar-slackin-drwasho.herokuapp.com/

M0Rf30 commented:
I'm on slack
